src/services/publishing/publishing.ts

```diff
@@ -95,13 +99,15 @@ export const publishTransferableRecordJob = async (job: PublishingJob, wallet: W
     { nonce }
   );
   const escrowDeploymentTx = await escrowDeploymentReceipt.wait();
+  // eslint-disable-next-line prettier/prettier
   const deployedTitleEscrowArgs = escrowDeploymentTx.events?.find(
     (event) => event.event === "TitleEscrowDeployed"
+    // eslint-disable-next-line prettier/prettier
```

Comment:
? why disable prettier for these 2 lines

Reply:
This two-line shows an error within the vsc and requires formatting. If I format the code, it will fail the eslint. Could you help me check if this issue happens on ur vsc as well ?

Reply:
just let the lint handle the formatting. (when you say format the code youre doing that manually right?)
npm lint fix handles both formatting and linting as we added that as a rule

Reply:
okay i will remove the disable prettier